mycli/main.py

```diff
@@ -730,6 +733,15 @@ def run_query(self, query, table_format=None, new_line=True):
             for line in output:
                 click.echo(line, nl=new_line)
 
+    def output_to_file(self, text):
+        output = special.get_redirected_output()
```

Comment:
This is the second time `get_redirected_output()` gets called (see line 596). Can we pass an `append` argument to this function so we don't have to call it again?